### ISSUE TITLE
Phalcon\Forms\Element::setLabel('0') render error

### ISSUE THREAD

**Issue by pwoszczyk** [phalcon/cphalcon#12148]
Phalcon\Forms\Element::setLabel($label) render incorrect if $label = '0'.

Example:
$field = new Numeric('value');
$field->setLabel('0');
$field->label() // <label for="value">value</label>
- Phalcon version: 2.0.13
- PHP Version: 5.6.11
- Operating System: Windows 7
- Installation type: precompiled dll
- Server: Apache

**Comment by pwoszczyk** [comment]
Same on:
- Phalcon version: 3.0.1
- PHP Version: 7.0.8
- Operating System: Ubuntu 16.04
- Installation type: packagecloud.io
- Server: Apache

**Comment by gguridi** [comment]
I think it's fair to be able to associate '0' as label, in this case it shouldn't validate as empty label.

I tried to use empty label as condition because zephir documentation says that ‘Empty’ means the expression is null, is an empty string or an empty array. But that's not true and '0'also validates as false. So the only way to add '0' as a label is by explicitly check if it's numeric as condition.

Added a pull request with the change.
